.ado/pipelines/templates/stages-full-release.yaml

```diff
@@ -310,11 +310,11 @@ stages:
     parameters:
       terraformWorkingDirectory: 'src/testing/loadtest-locust/infra'
       customPrefix: '$(prefix)$(customSuffix)'
-      numberOfWorkerNodes: 10 # number of locust worker nodes
-      testDurationSeconds: 300 # locust test duration
-      locustUser: 500 # locust number of simulated users
-      locustSpawnRate: 10 # locust spawn rate users / per second
-      locustHeadless: true # run locust in headless mode
+      numberOfWorkerNodes: $(loadTestNumberOfWorkerNodes)
+      testDurationSeconds: $(loadTestDurationInSeconds) # load test duration
```

Comment:
```suggestion
      testDurationSeconds: $(loadTestDurationInSeconds)
```

Reply:
why do you want to drop the comments here?

Reply:
because they are just redundant

Reply:
not within this file - we have comments for containerImageName, containerImageDockerFile and many other parameters in this file as well. not sure if i'd drop them here. i still think that using comments here adds value

Reply:
or do you think the parameter name itself is self-explaining?

Reply:
yeah I'd say so. I mean, feel free to leave them here. Its just another set of comments that might drift

Reply:
no, i agree - i think the variable names are self-explaining